packages/cli/src/cmds/preview/constants.ts

```diff
@@ -107,7 +107,9 @@ export const doctorResult = {
   NodeNotSupported: `Node.js (@CurrentVersion) is not in the supported version list (@SupportedVersions).`,
   NodeSuccess: `Supported Node.js version (@Version) is installed`,
   InstallNode:
-    "Go to https://nodejs.org/about/releases/ to install Node.js (recommended version v14)",
+    "Go to https://nodejs.org/about/releases/ to install Node.js (recommended version v16)",
+  BypassNode12: `To continue preview with Node.js v12, please follow the link (@Link) to disable Node.js check with TeamsFx CLI.`,
```

Comment:
What if user installs node 10 or any other earlier version? Is this message strictly for node 12 only?

Reply:
For node (10/8/6...), the only change is that this line will disappear. 
Node 12 is special because deprecated just last month. There are still some users stick on node 12.
But node 10 has been deprecated for over 1 year.

![image](https://user-images.githubusercontent.com/12523371/167799217-656808e7-14f7-408c-a98b-6fcf05eebd04.png)